### PR TITLE
Fix #5 added run.sh and doc

### LINKED ISSUE #5: update doc on running

### PR DESCRIPTION
**Pull request by robnagler**

### CODE REVIEW

File: etc/run.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+set -eou pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+_main() {
+    if [[ ! -d ~/mail ]]; then
+        echo ~/mail " needs to be created, see sirepo/etc" 1>&2
+        return 1
+    fi
+    exec docker run --rm -it --network=host \
+       -e SIREPO_E2E_EMAIL=vagrant+sirepo-e2e@localhost.localdomain \
```
Comment (e-carlin) [inline etc/run.sh:14]: Does the `+` work? It doesn't work for me when running `bash sirepo/etc/run-jupyterhub./sh` and logging in through the GUI

```
sirepo/uri_router.py:443:_call_api api=authEmailLogin exception={'vagrant+sirepo-e2e@localhost.localdomain': (550, b'5.1.1 <vagrant+sirepo-e2e@localhost.localdomain>: Recipient address rejected: User unknown in local recipient table')} stack=Traceback (most \
recent call last):
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/uri_router.py", line 434, in _call_api
    await getattr(qcall, qcall.uri_route.func_name)(**kwargs)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/auth/email.py", line 160, in api_authEmailLogin
    return _send_login_email(
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/auth/email.py", line 128, in _send_login_email
    if not _send_smtp(user_data, uri):
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/auth/email.py", line 134, in _send_smtp
    return sirepo.smtp.send(
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/smtp.py", line 33, in send
    _SEND(m)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/smtp.py", line 68, in _send_directly
    s.send_message(msg)
  File "/home/vagrant/.pyenv/versions/3.9.15/lib/python3.9/smtplib.py", line 986, in send_message
    return self.sendmail(from_addr, to_addrs, flatmsg, mail_options,
  File "/home/vagrant/.pyenv/versions/3.9.15/lib/python3.9/smtplib.py", line 901, in sendmail
    raise SMTPRecipientsRefused(senderrs)
smtplib.SMTPRecipientsRefused: {'vagrant+sirepo-e2e@localhost.localdomain': (550, b'5.1.1 <vagrant+sirepo-e2e@localhost.localdomain>: Recipient address rejected: User unknown in local recipient table')}
```

Comment (robnagler) [reply etc/run.sh:14]: Yes, it works. Is postfix setup on your vm?

**Review by gurhar1133 — APPROVED** [review]
looks good

File: etc/run.sh (hunk 1 above)
Comment (e-carlin) [reply etc/run.sh:14]: I thought so. But, apparently not correctly. I ran through the steps in https://github.com/radiasoft/sirepo/blob/3fa94dcbc7ec16897f12bdfb039eb3c0ce51b74a/etc/run-auth-moderate.sh and now it works. Sorry for the confusion.

**Pull request merged**